src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.GenerateAssemblyInfo.targets

```diff
@@ -45,11 +45,12 @@ Copyright (c) .NET Foundation. All rights reserved.
    -->
   <Target Name="GenerateAssemblyInfo"
           BeforeTargets="CoreCompile"
-          DependsOnTargets="PrepareForBuild;GetAssemblyVersion;CoreGenerateAssemblyInfo"
+          DependsOnTargets="PrepareForBuild;CoreGenerateAssemblyInfo"
           Condition="'$(GenerateAssemblyInfo)' == 'true'" />
 
   <Target Name="CoreGenerateAssemblyInfo"
           Condition="'$(Language)'=='VB' or '$(Language)'=='C#'"
+          DependsOnTargets="GetAssemblyVersion"
```

Comment:
The intent of having GetAssemblyVersion on GenerateAssemblyInfo and not here was to make it so that the AssemblyVersion is always available to CoreGenerateAssemblyInfo overrides for languages other than C# or VB.

Is moving this related to the feature requested here?